readme.md

```diff
@@ -1,62 +1,70 @@
-# Dog Search 
+# Image Search Application with Weaviate
 
 This project's origin is [here](https://github.com/weaviate/weaviate-examples/tree/main/nearest-neighbor-dog-search).
 
 ## Description
-In this demo, we will use Weaviate to create an image-search application. We'll develop a dog image-search application that lets users upload a photo of a dog and receive a list of dog breeds that are the most similar.
+This guide demonstrates how to build an Image Search Application with Weaviate. The application allows users to upload images of dogs and find the most similar dog breed. While we focus on dogs, you can adapt this approach for various image recognition and product search applications.
 
-(TODO: Add demo video)
```

Comment:
Let's keep this line as well, so future contributors know that there is a TO DO here.